### PR TITLE
BETA: Register mptamilan.is-a.dev

### PR DESCRIPTION
**Pull request by mptamilan**
Added `mptamilan.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Royal5074** [comment]
.

File: domains/mptamilan.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Royal5074",
+        "email": "whiteroyaldev@hotmail.com"
+    },
+    "record": {
+        "CNAME": "mptamilan.netlify.app"
```
Comment (DEV-DIBSTER) [inline domains/mptamilan.json:7]: We do not support netlify.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/mptamilan.json (hunk 1 above)
Comment (Royal5074) [reply domains/mptamilan.json:7]: Oh

Comment (Royal5074) [reply domains/mptamilan.json:7]: Delete the domain

**Pull request closed without merging**